SPIKES/Routing/go_router_example/lib/navigation/app_router.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:get_it/get_it.dart';
+import 'package:go_router_example/mock/product.dart';
+import 'package:go_router_example/pages/pages.dart';
+import 'package:go_router/go_router.dart';
+import 'package:go_router_example/state/app_state.dart';
+
+class AppRouter {
```

Comment:
Along with the normal routes, I would put another one with '/' that redirects to '/first' and maybe an error builder to show how it automatically handles bad URLs